package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gen3/ui-component",
-  "version": "0.6.6",
+  "version": "0.6.7",
```

Comment:
bump up minor version if you think this is a new feature, otherwise, keep this patch version but change what you have in PR description to `Improvements`

Reply:
updated to 0.7.0 also added to the storybook